src/utils/pools.tsx

```diff
@@ -398,12 +392,11 @@ export const swap = async (
       toAccount,
       pool.pubkeys.mint,
       pool.pubkeys.feeAccount,
+      hostFeeAccount,
       pool.pubkeys.program,
       programIds().token,
       amountIn,
-      minAmountOut,
-      hostFeeAccount,
-      isLatestSwap
```

Comment:
This argument was what directs to the latest instruction or the latest.
I propose to keep arguments organized as if we were calling `TokenSwap.swapInstruction` and add the extra `isLatestSwap` argument to branch the logic